src/index.js

```diff
@@ -19,17 +19,12 @@ const dir = config.directory;
 const youTube = new YouTube();
 youTube.setKey(config.youtube.apikey);
 
-const createDirectory = (name) => {
-  playlistName = name;
-  if (!fs.existsSync(playlistName))
```

Comment:
If this is removed what happens if a playlist is downloaded twice to the same directory? There is no check anymroe